site/content/docs/main/data-movement-backup-pvc-configuration.md

```diff
@@ -0,0 +1,46 @@
+---
+title: "BackupPVC Configuration for Data Movement Backup"
+layout: docs
+---
+
+`BackupPVC` is an intermediate PVC created by the exposer for VGDP (Velero Generic Data Path) to access data from during
+the data movement backup operation.
+
+In some scenarios users may need to configure some advanced options of the backupPVC so that the data movement backup
+operation could perform better.
+
+Velero introduces a new section in `node-agent-config` configMap called `backupPVC`, through which you can specify the following
```

Comment:
Might need to clarify here that name is configurable etc if https://github.com/vmware-tanzu/velero/pull/8097 merges.

Reply:
Done, updated